src/modules/blocks/tickets/capacity-table/template.js

```diff
@@ -56,7 +56,7 @@ const CapacityTable = ( {
 CapacityTable.propTypes = {
 	isSettingsLoading: PropTypes.bool,
 	independentCapacity: PropTypes.number,
-	sharedCapacity: PropTypes.string,
+	sharedCapacity: PropTypes.number,
```

Comment:
Does this break if the code returns a string? I mean, can we check since the JSON API might be returning a string.

Reply:
this doesn't break

https://www.loom.com/share/cb070094e4344b738572971f654edefe

I can still create tickets. In fact this prop is being used as number, not as string https://github.com/the-events-calendar/event-tickets/blob/master/src/modules/blocks/tickets/capacity-table/template.js#L27 from 5 years ago